fork/Microsoft.Health.FellowOakDicom.Serialization/Microsoft.Health.FellowOakDicom.Serialization.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
project name starts with `Microsoft.Health` so that can be signed automatically in workspace-platform